integration/test/app/workload/workload_cluster_test.go

```diff
@@ -106,7 +106,7 @@ func TestWorkloadCluster(t *testing.T) {
 				// Install app-operator in the workload cluster namespace.
 				AppCRNamespace: key.WorkloadClusterNamespace(),
 				CatalogName:    key.ControlPlaneTestCatalogName(),
-				Name:           project.Name(),
+				Name:           fmt.Sprintf("%s-%s", project.Name(), key.WorkloadClusterNamespace()),
```

Comment:
```suggestion
				AppCRName:           fmt.Sprintf("%s-%s", project.Name(), key.WorkloadClusterNamespace()),
				Name:                       project.Name(),
```

@tomahawk28 I think this should be AppCRName and Name should be just the project name as it maps to .spec.name in the app CR.

Reply:
You're right. I'm going to update this. Thanks!